integration/pmem/pmem_test.sh

```diff
@@ -50,6 +52,25 @@ function setup() {
 	fi
 }
 
+function ctr_exec {
+	args=()
+	for i; do
+		if [[ $i = *" "* ]]; then
+			args+=(\""${i}"\")
+		else
+			args+=("${i}")
+		fi
+	done
+
+	sleep 5
+	tmp_dir="$(mktemp -d)"
+	eval ctr t exec --exec-id $((exec_id++)) --cwd / --fifo-dir "${tmp_dir}" "${args[@]}"
+	ret=$?
```

Comment:
I don't think this will work if `ctr` fails as the script is running `set -e`. You could use the following (admittedly gross-looking but useful) hack:

```bash
{ eval ctr ... ; ret=$?; } || true

rm -rf "${tmp_dir}

return $ret
```